lightning_pose/models/backbones/vits.py

```diff
@@ -89,14 +89,6 @@ def load_vit_backbone_checkpoint(base, checkpoint: str):
     for key, value in ckpt_vit_pretrain.items():
         if key.startswith("vit_mae."):
             model_key = key.replace("vit_mae.vit.", "")
-            # Skip known problematic layers with size mismatches
-            if any(prob in model_key for prob in [
-                "position_embeddings",
-                "patch_embeddings.projection",  # in case backbone was trained with grayscale imgs
-                "decoder_pos_embed",
-                "decoder_pred",
-            ]):
```

Comment:
I think we should continue to skip `decoder_pos_embed` and `decoder_pred`, right?

Reply:
I guess the decoder keys won't be in the vision encoder state dict though, so not a problem!